product_docs/docs/pgd/5/rel_notes/pgd_5.5.0_rel_notes.mdx

```diff
@@ -0,0 +1,74 @@
+| BDR       | 5.5.0   | Log completion of an extension upgrade                                                                                  |           |
+| BDR       | 5.5.0   | Add restrictions for group commit options                                                                               |           |
+| BDR       | 5.5.0   | Ensure bdr.remove_commit_scope() handles non-existent commit scope                                                      |           |
+| BDR       | 5.5.0   | Execute each autopartition task in its own transaction                                                                  | RT101407  |
+| BDR       | 5.5.0   | Use DETACH CONCURRENTLY to drop partitions.                                                                             | RT101407  |
+| BDR       | 5.5.0   | Disallow node group creation on a node bad state                                                                        |           |
+| BDR       | 5.5.0   | Grant additional object permissions to role "bdr_read_all_stats"                                                        |           |
+| BDR       | 5.5.0   | Add support for readonly proxy routing                                                                                  |           |
+| BDR       | 5.5.0   | Improve stability of Raft consensus and workers by handling dynamic shared memory errors in the right place             |           |
+| BDR       | 5.5.0   | Report current manager activity in pg_stat_activity                                                                     |           |
+| BDR       | 5.5.0   | Expose number of changes processed by writer in a large transaction                                                     |           |
+| BDR       | 5.5.0   | Restart the replication connection for bdr_init_physical                                                                |           |
+| BDR       | 5.5.0   | Add replication parameter to connectdb()                                                                                |           |
+
+
+
+
+## Bug fixes
+
+| Component | Version | Description                                                                       | Addresses |
+|-----------|---------|-----------------------------------------------------------------------------------|-----------|
+| BDR       | 5.5.0   | Improve handling of node group configuration parameter "check_constraints"        | RT99956   |
+| BDR       | 5.5.0   | Prevent potential segfault in bdr.monitor_group_versions()                        | RT102290  |
+| BDR       | 5.5.0   | Correctly elect a new leader when the current leader gets route_writes turned off |           |
+| BDR       | 5.5.0   | Prevent unexpected writer terminations with improved queue flush process          | RT98966   |
+| BDR       | 5.5.0   | Fix multi-row conflict accidentally deleting the wrong tuple multiple times()                        |   |
```

Comment:
`multiple times()` seems like a typo or copy'n'pasta of some sort.

Reply:
Autocomplete (one day we'll not use mdx and things like this will stop.)

```suggestion
| BDR       | 5.5.0   | Fix multi-row conflict accidentally deleting the wrong tuple multiple times                       |   |
```